### PR TITLE
completed prep-html-syntax Jacob Yi

### PR DESCRIPTION
**Pull request by Yi-Jacob**
Screen Shot 2022-01-09 at 1 38 49 PM

### CODE REVIEW

File: prep-html-syntax/index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>
+    HTML Syntax
+  </title>
+
```
Comment (thebearingedge) [inline prep-html-syntax/index.html]: Be sure to close your `<head>` element before you open your `<body>` element.

File: prep-html-syntax/index.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+<body>
+
+
+  <h1 style="color: blue;">My favorite Star Wars Character is The Mandalorian(Din Djarin)</h1>
+  <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQDfASyn6WVGz0MczX53WRbwwO8LJUS_40K2g&usqp=CAU" alt="Mando">
+  <p>
+    He is a great protagonist who is able to use his wit and fighting skills to accomplish whatever his hear desires.
+  </p>
+</body></html>
+</DOCTYPE html>
```
Comment (thebearingedge) [inline prep-html-syntax/index.html]: There should only be one doctype declaration at the top of your document.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by Yi-Jacob** [comment]
Screen Shot 2022-01-09 at 2 41 02 PM

**Review by thebearingedge — CHANGES_REQUESTED** [review]
this can be removed
image

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**